src/lib/getTxLink.ts

```diff
@@ -45,6 +46,7 @@ export const getTxBaseUrl = ({
   stepSource,
   defaultExplorerBaseUrl,
   isOrder,
+  isBridge,
```

Comment:
Also not sure about bringing this general terminology here just for the sake of Butter, especially since bridging typically refers to something v. different (bridging the same asset across chains)

Reply:
Wait no, it should be `relayer`, not `relay` - https://github.com/shapeshift/web/pull/9833/commits/1c36ea6b825acd9f4b04a9bf243cd383d15bc716